.env.development

```
@@ -1 +1,2 @@
 NEXT_PUBLIC_STORYBOOK_URL=http://localhost:6006
+ALLOWED_DEV_ORIGINS=100.115.47.104
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLOWED_DEV_ORIGINS key should go before the NEXT_PUBLIC_STORYBOOK_URL key

(UnorderedKey)